app/templates/components/public/session-item.hbs

```diff
@@ -20,9 +20,11 @@
           </div>
         </div>
         <div class="ui right aligned column four wide less left padding">
-          {{#if session.startsAt}}
+          {{#if session.microlocation.name}}
```

Comment:
Leave this be, rest is fine.

Reply:
I guess by default there is a time set for the sessions, so by keeping `startsAt` in the if condition, it will always get true and the block will execute. That's why I shifted to microlocation name since on the session getting scheduled, the session will have a location and a time slot for sure.

Reply:
actually, for unscheduled sessions it is supposed to be null, seems weird that they will have a startTime by default, maybe open an issue for that too. It _should_ be null.

Reply:
Opened #2110 to set the unscheduled sessions start and end time to null.